internal/promoter/image/impl.go

```diff
@@ -52,9 +52,15 @@ type DefaultPromoterImplementation struct {
 }
 
 // NewDefaultPromoterImplementation creates a new DefaultPromoterImplementation instance.
-func NewDefaultPromoterImplementation() *DefaultPromoterImplementation {
+func NewDefaultPromoterImplementation(opts *options.Options) *DefaultPromoterImplementation {
+	signOpts := sign.Default()
+	signOpts.CertIdentity = opts.SignCheckIdentity
+	signOpts.CertOidcIssuer = opts.SignCheckIssuer
+	signOpts.CertIdentityRegexp = ""
+	signOpts.CertOidcIssuerRegexp = ""
```

Comment:
Nit: since we set those parameters in multiple places, we could create a function that's going to return signOpts with those parameters set. That way we avoid potential drift, e.g. if we forget to update in some place.

Reply:
sgtm, do you want to open a PR for that? thanks!

Reply:
Sure, sounds good

Reply:
Looks like this is now fixed. 🙂